WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosPageable.swift

```diff
@@ -0,0 +1,48 @@
+
+struct StockPhotosPageable: Pageable {
+    let number: Int
+    let pageHandle: Int
+
+    static let defaultPageSize = 40
+    static let defaultPageIndex = 0
+
+    func next() -> Pageable? {
+        if pageHandle == 0 {
+            return nil
+        }
+
+        return StockPhotosPageable(number: number, pageHandle: pageHandle)
+    }
+
+    func pageSize() -> Int {
+        return number
+    }
+
+    func pageIndex() -> Int {
+        return pageHandle
+    }
+}
+
+extension StockPhotosPageable {
+    static func initial() -> StockPhotosPageable {
```

Comment:
This was intended as a factory to create the initial pageable, so that it conveys some more meaning than just an initialiser. That's why I lean towards making it static, it, to me, suggests that it is an special method, not the regular way to initialise the struct, but the one special case when we want to initialise it with a set of default values. What do you think?

Reply:
Sounds good! I see your point.
Let's put a small method doc to state the default values?